extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -518,60 +542,76 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.data.savedSvgFileName.length > 0);
       };
 
+      var getSize = function() {
+        var size = ctrl.fabricjsOptions.size;
+        // Removes the word "px" from the end of the string and converts
```

Comment:
How does the below line remove "px" exactly? Is the below code not just returning a size?

Reply:
[Parse int ](https://stackoverflow.com/questions/4860244/how-to-delete-px-from-245px)removes any alphabets and converts it into int.